docs/contributing/mac-deps-setup.md

```diff
@@ -0,0 +1,40 @@
+# Mac System Dependencies
+
+## Node
+
+There are two approaches to installing Node, **simple** and **version managed**. We recommend the version managed route, but the simple method will work fine if you don't have node installed on your machine and just want to get running!
+
+### Version Managed
+
+We recommend using [nvm](https://github.com/creationix/nvm) to manage your node version. This makes it easier to know you are running the correct version of node for this project without effecting setup for other projects on your machine.
+
+- Install nvm from https://github.com/creationix/nvm#install-script
+- Install the version of node for GitHub Desktop with `nvm install`
```

Comment:
Do we need to indicate that these commands should be run within the clone of the repository?